hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -3028,11 +2993,11 @@ private void validate() {
       Objects.requireNonNull(writeConfig.getString(BASE_PATH));
       if (writeConfig.isEarlyConflictDetectionEnable()) {
         checkArgument(writeConfig.getString(WRITE_CONCURRENCY_MODE)
-                .equalsIgnoreCase(WriteConcurrencyMode.OPTIMISTIC_CONCURRENCY_CONTROL.value()),
+                .equals(WriteConcurrencyMode.OPTIMISTIC_CONCURRENCY_CONTROL.name()),
```

Comment:
Same here, could we ignore case as before?

Reply:
Addressed